man/tpm2_activatecredential.1.md

```diff
@@ -33,7 +33,7 @@ These options control the object verification:
 
   * **-P**, **--password**=_PASSWORD_:
     Use _PASSWORD_ for providing an authorization value for the _KEY\_HANDLE_.
-    Passwords should follow the "password formatting standards, see section "Password Formatting".
+    Passwords should follow the "authorization formatting standards, see section "Authorization Formatting".
```

Comment:
Missing enclosing `"` after standards.

Reply:
Or probably just remove the opening `"` before authorization.

Reply:
Yeah I was removing the ", missed that one.